docs/api-testing/use-cases/fact.md

```diff
@@ -0,0 +1,300 @@
+---
+id: fact
+title: Using Fact for Handling Email Notifications
+sidebar_label: Improving Email Notifications using Fact
+description: Using Fact for Handling Email Notifications
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The **Fact** component is used to control the behaviour of Email notifications, which (if enabled) alert you to test failures. It enables you to add information about the nature of the test execution.
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+- An existing API Testing Project. For details on how to create one, see [API Testing Quickstart](/api-testing/quickstart/).
+- Your email address already saved in the [Notifications](/api-testing/project-access/#email-notifications) section.
+- Familiarity with the [API Testing Composer](/api-testing/composer/).
+
+## Setting Alert Environments
+
+Testing activity is tracked using a test ID number. This may not work if you're testing in multiple environments, as an incident could be environment-specific. When a Fact component is added to a test, it will inform our system which environment the execution relates to so that the incident signature will carry the environment as well.
+
+Assume that you are running the same test in _staging_ and _production_, you will get an email notification the first time the test fails in one of the two environments. This does not allow you to know which environment the test fails in. The same will happens when the test starts working again.
+
+By configuring a **Fact** (together with **Tag**), you can add the environment value to the incident signature, and from that moment on the signature of the incident will be `id_of_the_test` + `value_of_environment`.
+In this way, you will get a separate notification for each environment the test fails in: for example, you will receive start/end incidents for `test123` in the production environment, and start/end incidents for `test123` in the staging environment, as a separate flow of events.
+
+Assume that in the variable scope of your test, you have a variable called _env_ that contains your environment string (_production, staging, qa_ etc.).
+
+By configuring a Fact in the following way, you can add the environment value to the incident signature:
+
+<img src={useBaseUrl('img/api-testing/fact.png')} alt="fact component" />
```

Comment:
This screen shot is missing.
```suggestion
<img src={useBaseUrl('img/api-testing/fact.png')} alt="fact component" />
```

Reply:
I used an existing image, so it's not included in the commit, but it should be available in the doc